openfecwebapp/templates/candidates-single-new.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'layouts/main.html' %}
+{% import 'macros/cycle-select.html' as select %}
+{% import 'macros/tabs.html' as tabs %}
+{% import 'macros/page-header.html' as header %}
+
+{% block title %}
+    {{ name }} - Candidate overview
+{% endblock %}
+
+{% set breadcrumbs=[('', 'Candidate profiles')] %}
+
+{% block body %}
+
+{{ header.header(name, breadcrumbs) }}
+
+<div class="u-padding--left u-padding--right tab-interface">
+  <header class="main">
+    <h1 class="entity__name">{{ name }}</h1>
+    <div class="heading--section">
+      <span class="t-data t-bold entity__type">Candidate for {{ office_full }}</span>
+      {% if office == 'S' %}<span class="t-data t-bold entity__type">{{ constants.states[state] }}</span>{% endif %}
```

Comment:
This should also have a check for `if office == 'H'` and show the state + district (for house candidates).